### ISSUE TITLE
complete read me for PCF

### ISSUE THREAD

**Issue by venki3v** [spring-cloud-stream-app-starters/tensorflow#20]
Is it possible to have a detailed read me document to run this on pivotal including how to do the shell and the applications?

**Comment by sabbyanandan** [comment]
Hi, @venki3v. We keep the App specific README's more generic to have it out of platform-specific dependencies. Have you looked at the [SCDF samples](https://github.com/spring-cloud/spring-cloud-dataflow-samples), though? We have PCF specific instructions [see: [`http | cassandra`](https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/htmlsingle/#http-cassandra-cf)] with a few e2e samples; perhaps that'd help. 

Once when you have SCDF running on PCF, you'd then create and deploy streams, and that's about it. For example, a Twitter-sentiment Analytics stream definition (see below) would look similar to [Twitter-Analytics](https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/htmlsingle/#spring-cloud-data-flow-samples-twitter-analytics-overview), except the Apps would be deployed to PCF.

> stream create foo --definition "twitterstream --consumerKey=<CONSUMER_KEY> --consumerSecret=<CONSUMER_SECRET> --accessToken=<ACCESS_TOKEN> --accessTokenSecret=<ACCESS_TOKEN_SECRET> | filter --expression=#jsonPath(payload,'$.lang')=='en' | twitter-sentiment --vocabulary=http://dl.bintray.com/big-data/generic/vocab.csv --model-fetch=output/Softmax --model=http://dl.bintray.com/big-data/generic/minimal_graph.proto | field-value-counter --field-name=sentiment --name=sentiment"

Give it a go; if you have any questions, please post them in StackOverflow or Gitter. Please also consider contributing an e2e sample on PCF to SCDF samples repo.
